docs/configuration/cluster_manager/cluster.rst

```diff
@@ -168,11 +168,33 @@ outlier_detection
   max_ejection_percent
     The maximum % of an upstream cluster that can be ejected due to outlier detection. Defaults to 10%.
 
-  .. _config_cluster_manager_cluster_outlier_detection_enforcing:
+  .. _config_cluster_manager_cluster_outlier_detection_enforcing_consecutive_5xx:
 
-  enforcing
-    The % chance that a host will be actually ejected when an outlier status is detected. This setting
-    can be used to disable ejection or to ramp it up slowly. Defaults to 100.
+  enforcing_consecutive_5xx
+    The % chance that a host will be actually ejected when an outlier status is detected through
+    consecutive 5xx. This setting can be used to disable ejection or to ramp it up slowly. Defaults to 100.
+
+  .. _config_cluster_manager_cluster_outlier_detection_enforcing_sr:
+
+  enforcing_sr
+    The % chance that a host will be actually ejected when an outlier status is detected through
+    success rate statistics. This setting can be used to disable ejection or to ramp it up slowly.
+    Defaults to 100.
+
+  .. _config_cluster_manager_cluster_outlier_detection_significant_host_threshold:
+
+  significant_host_threshold
```

Comment:
not clear this is related to request volume and success rate. Use a more specific name.